mpi/inner_script.sh

```diff
@@ -9,13 +9,16 @@ die () {
 BATCH_NNODES=$(flux resource list -n -o {nnodes})
 BATCH_NCORES=$(flux resource list -n -o {ncores})
 COMPILER=$1
-MPI=$2
+MPI=$(echo $2 | sed "s/\///")
 export NAME="$COMPILER"_"$MPI"
 
 test -n $COMPILER || die "COMPILER (argument 1) not set"
 test -n $MPI || die "MPI (argument 2) not set"
+if [[ $MPI == "openmpi5" ]]; then
```

Comment:
Would it be simpler to avoid removing the first `/` from the MPI name and instead just have 
```sh
if [[ $MPI == "openmpi/5" ]]; then
```

This would also address the comment below regarding `module load $2`